rest-api/src/main/resources/definitions/abstract_study_participant.yml

```diff
@@ -93,4 +93,8 @@ properties:
         description: The identifier of the organization this account is a member of (this will be an administrative account and not a study participant). Once set, it cannot be changed on an update (there are separate organization membership APIs to change organizational membership).
     note:
         type: string
-        description: Optional note related to user account. Can only be edited and viewed by an administrative account.
\ No newline at end of file
+        description: Optional note related to user account. Can only be edited and viewed by an administrative account.
+    clientTimeZone:
+        type: string
+        description: |
+            Participant's time zone. Can be updated or deleted. Must be an IANA time zone name (eg. "America/Los_Angeles").
```

Comment:
Use the example field instead of including "America/Los_Angeles" in the description field.